dist/index.html

```diff
@@ -11,8 +11,28 @@
     <title>TV-Shows</title>
   <script defer src="%5Bmain%5D.js"></script></head>
   <body>
+    <header>
+      <div class="container flex-spaced">
+        <div class="logo">logo</div>
+        <div id="comment-section"></div>
+
```

Comment:
Please remove unnecessary white spaces form your HTML. 👍 To improve readability.